### PR TITLE
fix: restore coupon in order summary, e2e tests (GH-5154)

### PR DESCRIPTION
**Pull request by mazamcodes**
Closes: #5154

### CODE REVIEW

File: projects/storefrontlib/src/cms-components/cart/cart-shared/order-summary/order-summary.component.html [hunk 1]
```diff
@@ -53,3 +53,8 @@ <h4>{{ 'orderCost.orderSummary' | cxTranslate }}</h4>
   [promotions]="cart.appliedOrderPromotions"
 ></cx-promotions>
 -->
+
+<cx-applied-coupons
```
Comment (LTiger14) [inline projects/storefrontlib/src/cms-components/cart/cart-shared/order-summary/order-summary.component.html]: Why did you add this here? Isn't there a feature flag or level or something for this? Also you need to add a trailing white line for prettier.

Comment (mazamcodes) [reply projects/storefrontlib/src/cms-components/cart/cart-shared/order-summary/order-summary.component.html]: Adding this back because removing it broke the e2e tests, but more importantly, removed all mention of coupon from order summary.
